greendao/src/main/java/com/epam/greendao/GreenDAO.java

```diff
@@ -0,0 +1,106 @@
+package com.epam.greendao;
+
+import android.content.Context;
+
+import com.epam.benchmark.IEntity;
+import com.epam.benchmark.IStorage;
+
+import java.io.IOException;
+import java.util.List;
+
+import de.greenrobot.dao.query.QueryBuilder;
+import de.greenrobot.dao.query.WhereCondition;
+import de.greenrobot.daogenerator.DaoGenerator;
+import de.greenrobot.daogenerator.Entity;
+import de.greenrobot.daogenerator.Schema;
+
+
+/**
+ * Created by shulha_dmytro on 1.10.15.
+ */
+public class GreenDAO implements IStorage {
+
+    public static final String GREENDAO = "greendao";
+    private ModelDao dao;
+
+    public static void main(String[] args) {
```

Comment:
Move it to separate helper class! The method is designed to be invoked only once in his life